health/notifications/pagerduty/README.md

```diff
@@ -1,46 +1,69 @@
 <!--
-title: "PagerDuty"
+title: "Send alert notifications to PagerDuty"
+description: "Send alerts to your PagerDuty dashboard any time an anomaly or performance issue strikes a node in your infrastructure."
+sidebar_label: "PagerDuty"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notifications/pagerduty/README.md
 -->
 
-# PagerDuty
+# Send alert notifications to PagerDuty
 
-[PagerDuty](https://www.pagerduty.com/company/) is the enterprise incident resolution service that integrates with ITOps and DevOps monitoring stacks to improve operational reliability and agility. From enriching and aggregating events to correlating them into incidents, PagerDuty streamlines the incident management process by reducing alert noise and resolution times.
+[PagerDuty](https://www.pagerduty.com/company/) is the enterprise incident resolution service that integrates with ITOps
+and DevOps monitoring stacks to improve operational reliability and agility. From enriching and aggregating events to
+correlating them into incidents, PagerDuty streamlines the incident management process by reducing alert noise and
+resolution times.
 
-Here is an example of a PagerDuty dashboard with Netdata notifications:
+Here's an example of a PagerDuty dashboard with Netdata alert notifications:
 
-![PagerDuty dashboard with Netdata notifications](https://cloud.githubusercontent.com/assets/19278582/21233877/b466a08a-c2a5-11e6-8d66-ee6eed43818f.png)
+![PagerDuty dashboard with Netdata alert
+notifications](https://user-images.githubusercontent.com/1153921/118317133-872a4100-b4ac-11eb-9cf1-70414aba010f.png)
 
-To have Netdata send notifications to PagerDuty, you'll first need to set up a PagerDuty `Generic API` service and install the PagerDuty agent on the host running Netdata.  See the following guide for details:
+## What you need to get started
 
-<https://www.pagerduty.com/docs/guides/agent-install-guide/>
+- An installation of the open-source [Netdata](/docs/get-netdata.mdx) monitoring agent
+- An installation of the [PagerDuty agent](https://www.pagerduty.com/docs/guides/agent-install-guide/) on the node
+  running Netdata
+- A PagerDuty `Generic API` service using either 
```

Comment:
Is there supposed to be more to this sentence? I might be missing something " A PagerDuty `Generic API` service using either"

Reply:
Good catches on both of these! Fixed now.